wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/DIOSim.java

```diff
@@ -151,6 +164,7 @@ public CallbackStore registerIsInputCallback(NotifyCallback callback, boolean in
    *
    * @return true if Input
    */
+  @Override
```

Comment:
All these methods would be inherited, no need to override them. The deprecated stub needs only constructors.